### ISSUE TITLE
Support for style block in new datadog_dashboard

### ISSUE THREAD

**Issue by saipranav** [terraform-providers/terraform-provider-datadog#275]
Hi Contributors,

### Terraform Version
terraform version 0.11.14
provider.datadog: version = "~> 2.1"

### Affected Resource(s)
Please list the resources as a list, for example:
- datadog_dashboard

### Terraform Configuration Files
```hcl
widget {
    timeseries_definition {
      title = "number of consul per dc, prod(red) & dorp(blue)"

      request {
        q            = "count:system.uptime{hardware_env:prod,$region} by {dc}"
        display_type = "line"

        style {
          palette = "warm"
        }
      }

      request {
        q            = "count:system.uptime{hardware_env:dorp,$region} by {dc}"
        display_type = "line"

        style {
          palette = "cool"
        }
      }
    }
  }
```

### Expected Behavior
style block to color the lines are not present in new `datadog_dashboard` but it was available in `datadog_timeboard`
https://www.terraform.io/docs/providers/datadog/r/timeboard.html#nested-graph-style-block

### Actual Behavior
widget.0.timeseries_definition.0.request.0: invalid or unknown key: style

Am I missing some information to bring colors into line graphs?

I would like to have this feature if there is a feasibility!

Thanks to all contributors

### References
#211

**Comment by bkabrda** [comment]
CC @enbashi - Rami, is this something that we could make work?

**Comment by enbashi** [comment]
@bkabrda yes.  I am working on a fix to address both this and  #271.
